README.md

```diff
@@ -51,14 +51,14 @@ documentation].
 
 | Release |  Current Patch  | Release Date |      EOL      |
 |:-------:|:---------------:|:------------:|:-------------:|
-|   v1.1  |     [v1.1.4]    |  Mar 3, 2021 |  August 2021  |
 |   v1.2  |     [v1.2.4]    | Apr 27, 2021 | October 2021  |
 |   v1.3  |     [v1.3.1]    | Jun 29, 2021 | December 2021 |
-|   v1.4  |     Upcoming    | Aug 31, 2021 | February 2022 |
+|   v1.4  |     [v1.4.0]    | Aug 31, 2021 | February 2022 |
+|   v1.5  |     Upcoming    | Oct 26, 2021 | April 2022 |
```

Comment:
nitpick:
```suggestion
|   v1.4  |     [v1.4.0]    | Aug 31, 2021 | February 2022 |
|   v1.5  |     Upcoming    | Oct 26, 2021 |   April 2022  |
```

Reply:
It shows up as centered in preview actually and I see others lean to left.

Reply:
Hmm, I think that must have happened in a recent update -- can we keep column width consistent?

Reply:
Done!